eslint.config.mjs

```diff
@@ -163,6 +213,69 @@ export default tseslint.config(
       'no-var': 'error',
       'no-void': 'error',
       'no-with': 'error',
+      'perfectionist/sort-enums': [
+        'error',
+        {
+          ...perfectionistCommonConfig.partition,
+        },
+      ],
+      'perfectionist/sort-interfaces': [
+        'error',
+        {
+          ...perfectionistCommonConfig.partition,
+        },
+      ],
+      'perfectionist/sort-jsx-props': [
+        'error',
+        {
+          customGroups: {
+            key: '^key$',
+            as: '^as$',
+            id: '^id$',
+            name: '^name$',
+            ref: '^ref$',
+            methods: '^on[A-Z].*$',
+          },
+          groups: ['key', 'as', 'id', 'name', 'ref', 'unknown', 'methods'],
```

Comment:
I’m not sure this helps to scan the code. I understand the selection of ‘special’ props that could go first, but it also requires people to remember which ones they are. I suggest we remove these exceptions and just sort everything alphabetically. What do you think @alimpens?

Reply:
Yeah, I agree, it's easier to find props / attrs when they're all sorted alphabetically.